test/jsdom-global-register.js

```
@@ -0,0 +1 @@
+module.exports = {};
```

ESLint

[error] 'module' is not defined.

(no-undef)